ursina/physics.py

```diff
@@ -1,34 +1,34 @@
 from panda3d.core import BitMask32, TransformState
 from panda3d.bullet import BulletRigidBodyNode, BulletPlaneShape, BulletBoxShape, BulletSphereShape, BulletCapsuleShape, XUp, YUp, ZUp, BulletTriangleMesh, BulletTriangleMeshShape, BulletDebugNode, BulletWorld
-from ursina.vec3 import Vec3
-from ursina.entity import Entity
-from ursina import scene
 from ursina.scripts.property_generator import generate_properties_for_class
-import time
+from ursina import Entity, scene, time, Vec3
 
 @generate_properties_for_class()
 class PhysicsHandler(Entity):
     def __init__(self):
+        super().__init__()
+        self._debugNode = BulletDebugNode('Debug')
+        self._debugNode.showWireframe(True)
+        self._debugNode.showConstraints(True)
+        self._debugNode.showBoundingBoxes(False)
+        self._debugNode.showNormals(False)
+        self.debug_node_path = scene.attachNewNode(self._debugNode)

```

Comment:
I've left it like that because it's possible to call the node_path of the RigidBody class, for example, and to be consistent with the rest of the code. The user might want to access it, on the other hand, to access the "node" attached to it and change certain parameters such as "showNormals".